app/config.py

```diff
@@ -533,7 +533,6 @@ class Staging(Config):
 
 
 class Live(Config):
-    NOTIFY_EMAIL_DOMAIN = os.environ.get('NOTIFY_EMAIL_DOMAIN')
```

Comment:
Where is this being set now? My understanding (which could be wrong, I'm a python noob) is that the prod version will basically set everything in the `Config` class, and then override anything in `Live`. Now neither of those areas have a setting for `NOTIFY_EMAIL_DOMAIN`

Reply:
Shows you what I get for pushing code during meetings. Meant to directly set notify.sandbox.10x... in `Config` and set the env with a default in `Development`. Should be fixed now.